acra/security-controls/transport-security/_index.md

```diff
@@ -2,3 +2,6 @@
 title: Transport security
 bookCollapseSection: true
 ---
+
```

Comment:
```suggestion
**TODO: Expand**
```
Some marker to `grep` before publishing. But will be better to add comments in the appropriate PR #96 and @iamnotacake can apply before merging.

Reply:
Same here - this is indeed marker for grep. 

If you want it to be more explicit - suggest ;)